### PR TITLE
chore(hybrid-cloud): Add generic docs for SCIM/SAML when renaming an org.

### PR DESCRIPTION
**Pull request by dashed**
Add blurbs in documentation pages for SAML and SCIM on warning users to update relevant configuration values when an organization slug name has been updated.

### CODE REVIEW

File: src/docs/product/accounts/sso/azure-sso.mdx [hunk 1]
```diff
@@ -6,6 +6,12 @@ description: "Set up Azure Active Directory single sign-on (SSO) on Sentry."
 
 ## Installation
 
+<Alert level="warning">
+
+If you rename your organization's slug, you will need to replace the previous organization slug with the new slug name in the relevant steps where you enter the SAML configuration values.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/azure-sso.mdx:11]: Edited the wording for style and concision and to remove what seems like some redundancy in the wording. Same edit applies to all of the notes.

```suggestion
If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.
```

**Review by imatwawana — APPROVED** [review]
I've made edits on one instance of the note, but it applies to all of them. Otherwise, looks good!

File: src/docs/product/accounts/sso/azure-sso.mdx [hunk 2]
```diff
@@ -6,6 +6,12 @@ description: "Set up Azure Active Directory single sign-on (SSO) on Sentry."
 
 ## Installation
 
+<Alert level="warning">
+
+If you change your organization slug, you'll need to make the same update in the steps where you enter the SAML configuration values.
```
Comment (imatwawana) [inline src/docs/product/accounts/sso/azure-sso.mdx:11]: @dashed - I was indicating that the change I suggested here should be applied in all the files.

Comment (dashed) [reply src/docs/product/accounts/sso/azure-sso.mdx:11]: @imatwawana yep! I'm applying them everywhere 👍

**Pull request merged**